src/modules/imageresizer/ui/Views/InputPage.xaml

```diff
@@ -130,9 +130,12 @@
                     <ColumnDefinition Width="Auto"/>
                     <ColumnDefinition Width="Auto"/>
                 </Grid.ColumnDefinitions>
-                
-                <TextBlock VerticalAlignment="Center"
-                           Visibility="{Binding ShowAdvancedSettings, Converter={StaticResource BoolValueConverter}}">
+
+                <TextBlock x:Name="AdvancedSettingsTextBlock"
+                           VerticalAlignment="Center"
+                           Visibility="{Binding ShowAdvancedSettings, Converter={StaticResource BoolValueConverter}}"
+                           IsEnabled="{Binding Visibility,ElementName=AdvancedSettingsTextBlock, Converter={StaticResource VisibilityBoolConverter}}"
```

Comment:
It would be interesting to understand why binding to `ShowAdvancedSettings` didn't work. But we have to move on since this UI may change completely so no point to spent time investigating it now.